styles/styles.css

```diff
@@ -382,4 +382,44 @@ main .section.highlight {
       max-width: 1279.98px;
       padding: var(--qui-spacing-0);
   }
-}
\ No newline at end of file
+}
+
+main > .section.two-columns {
+    padding-bottom: 0;
+    margin-bottom: 0;
+}
+
+main > .section.two-columns > div {
+    display: flex;
+    flex-direction: column;
+}
+
+@media (width >= 900px) {
+    main > .section.two-columns {
+        flex-direction: row;
+        display: flex;
```

Comment:
I think this looks correct with the current content.
I could see a problem is if somebody would use `two-columns` with more items in the section.
One option would be to simply say that a two-column sections only work with two child items.
Another option would be two try to use `display: grid` with `grid-template-columns: 2`. That way, no matter how many items are in a section, when using `two-columns`, they will always be displayed in two columns.
Up to you, it's just a suggestion.

Reply:
Thanks, applied.

Reply:
Something looks broken now:
<img width="1728" alt="Screenshot 2024-09-24 at 16 07 26" src="https://github.com/user-attachments/assets/e1cfd8a3-e38a-4a86-83ca-bc798b1fc76f">

Reply:
@andreituicu good catch, fixed.